stark-brainfuck/src/xfri.rs

```diff
@@ -727,7 +721,13 @@ mod xfri_tests {
         let fri: Fri<Hasher> =
             get_x_field_fri_test_object(subgroup_order, expansion_factor, colinearity_check_count);
         let mut proof_stream: StarkProofStream<Hasher> = StarkProofStream::default();
-        let subgroup = fri.domain.omega.get_cyclic_group_elements(None);
+        let subgroup: Vec<XFieldElement> = fri
+            .domain
+            .omega
+            .get_cyclic_group_elements(None)
+            .iter()
+            .map(|x| x.lift())
+            .collect();
```

Comment:
This allocates the entire FRI domain twice in memory.

Maybe this does not matter, since it's tutorial code.

Reply:
I agree that it doesn't matter, not right now at least.